packages/html-elements/tsconfig.json

```diff
@@ -1,8 +1,8 @@
-// @generated by expo-module-scripts
```

Comment:
Removing this means that this tsconfig will no longer be autogenerated, which isn't ideal. For most packages, I'd expect the "node" types not to be needed since the packages are for clients. Is there something Node-specific in this package?

Reply:
Yes, I'm not too familiar with the `html-elements` package but it has a `process.env` check inside View.tsx, which throws a `Cannot find name 'process'` error without "node" types.

https://github.com/expo/expo/blob/2a7b8cd031384e998226f906106481945d61eead/packages/html-elements/src/primitives/View.tsx#L150-L153

cc @EvanBacon

Reply:
Hm, we might just need to provide a stub type declaration for `process.env`. We shouldn't use all the Node type declarations here.

What's going on here is `process.env.NODE_ENV` is used to tell Metro, "please replace me with the string 'development' or 'production' based on the mode Metro is running in". Here, `process` has nothing to do with the app's process and `env` has nothing to do with the app's environment variables.

Reply:
Got it, updated it to use a stub type declaration